prime-router/metadata/schemas/hhsprotect/hhsprotect-covid-19.schema

```diff
@@ -256,3 +256,7 @@ elements:
     csvFields: []
   - name: pregnant_text
     csvFields: []
+
+  - name: processing_mode_code
+    default: T
+    csvFields: [{name: processingModeCode}]
```

Comment:
Interestingly, because hhsprotect schema extends: direct-covid-19, it already had this field - so this is just making it explicit.